### DISCUSSION TITLE
EU time (24 hours) and seconds support

### DISCUSSION THREAD

**Discussion by No0negit** [Tyrrrz/DiscordChatExporter#808] [General]
It'll be good to have a switch in interface, to have a 24 hours system / 12 hours system for people from different places of the world. It'll be helpful when setting a time limit.
Also, it'll be good to have seconds supported for more control over time limits.

**Comment by Tyrrrz** [comment]
See the wiki

**Reply by No0negit** [reply]
I'm talking not about exported channels, i'm talking about UI support for 24 hours and seconds, when you're setting time limit.

**Reply by Tyrrrz** [reply]
Ah I see. I'd have to look into that.
